### PR TITLE
fix tomli dependency,

### PR DESCRIPTION
**Pull request by PythonicNinja**
related to https://github.com/pypa/setuptools_scm/issues/608

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -28,6 +28,7 @@ classifiers =
 packages = find:
 install_requires =
     setuptools>=45
+    tomli~=1.0.0
```
Comment (RonnyPfannschmidt) [inline setup.cfg]: i believe by now the pin has to be corrected to `tomli~=1.0`

**Review by RonnyPfannschmidt — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (SebastianRehfeldt) [inline setup.cfg]: Is it required to keep tomli in `[options.extras_require]` or should it be removed there?

Comment (PythonicNinja) [reply setup.cfg]: ok changing 👍

Comment (PythonicNinja) [reply setup.cfg]: @RonnyPfannschmidt done

Comment (PythonicNinja) [reply setup.cfg]: I would say that if it's required we should also remove it from `extras_require`

Comment (PythonicNinja) [reply setup.cfg]: done

Comment (RonnyPfannschmidt) [reply setup.cfg]: actually it should stay only in the extra, and the helper to load it should error if its not found only and only if a `tool.setuptools_scm` section is actually in `pyproject.toml`

**Comment by RonnyPfannschmidt** [comment]
unfortunately i'll be away from the computer for about 1-2 hours now for childcare, see you later

**Review by RonnyPfannschmidt — APPROVED** [review]

**Comment by RonnyPfannschmidt** [comment]
since pip is used for install anyway, lets go for a hard dependency anyway, i'll update the docs and do a release afterwards

**Pull request merged**